client/consensus/babe/src/aux_schema.rs

```diff
@@ -16,23 +16,23 @@
 // You should have received a copy of the GNU General Public License
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
 
-//! Schema for BABE epoch changes in the aux-db.
+//! Schema for BABE session changes in the aux-db.
 
 use codec::{Decode, Encode};
 use log::info;
 
-use crate::{migration::EpochV0, Epoch};
+use crate::{migration::SessionV0, Session};
 use sc_client_api::backend::AuxStore;
-use sc_consensus_epochs::{
-	migration::{EpochChangesV0For, EpochChangesV1For},
-	EpochChangesFor, SharedEpochChanges,
+use sc_consensus_sessions::{
+	migration::{SessionChangesV0For, SessionChangesV1For},
+	SessionChangesFor, SharedSessionChanges,
 };
 use sp_blockchain::{Error as ClientError, Result as ClientResult};
 use sp_consensus_babe::{BabeBlockWeight, BabeGenesisConfiguration};
 use sp_runtime::traits::Block as BlockT;
 
-const BABE_EPOCH_CHANGES_VERSION: &[u8] = b"babe_epoch_changes_version";
-const BABE_EPOCH_CHANGES_KEY: &[u8] = b"babe_epoch_changes";
+const BABE_EPOCH_CHANGES_VERSION: &[u8] = b"babe_session_changes_version";
+const BABE_EPOCH_CHANGES_KEY: &[u8] = b"babe_session_changes";
```

Comment:
This is not backwards compatible any existing node will stop working after this. If we want to change the key name then we need to migrate.